app-crypt/swtpm/files/swtpm-0.7.2-Conditionalize-test-dependencies.patch

```diff
@@ -0,0 +1,55 @@
+From 71588da70f7cf7c8f946b80b4d93bc398af8498e Mon Sep 17 00:00:00 2001
+From: Christopher Byrne <salah.coronya@gmail.com>
+Date: Mon, 28 Mar 2022 22:10:07 -0500
+Subject: [PATCH] configure.ac: Conditionalize test dependencies
+
+Signed-off-by: Christopher Byrne <salah.coronya@gmail.com>
+---
+ configure.ac | 25 +++++++++++++++++--------
+ 1 file changed, 17 insertions(+), 8 deletions(-)
+
+diff --git a/configure.ac b/configure.ac
+index e9f9d8f..c7dc3bf 100644
```

Comment:
Awesome! Thanks for making this change. Could I ask you scrub the patch as outlined in the [Clean patch howto](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto)?

Reply:
OK, the headers generated by `git send-email` aren't necessary either though nor is the diffstat. Basically everything from `--- a/configure.ac` and up can be removed.

Reply:
OK, all removed

Reply:
I slightly disagree in that authorship is useful as is the subject, but yes, scrubbing is good.

Reply:
I agree, I wasn't very precise here. Including a brief [patch description](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#patch-descriptions) above the diff is encouraged :-)